### DISCUSSION TITLE
quasar 2, modifiable index.js?

### DISCUSSION THREAD

**Discussion by bmansfie** [quasarframework/quasar#9223] [CLI - SSR mode] [answered]
SSR mode appears to only have a generated production server file in .quasar/ssr-prod-webserver.js which has the following at the top.  

```
/**
 * THIS FILE IS GENERATED AUTOMATICALLY.
 * DO NOT EDIT.
 **/
```
 
The old src-ssr/index.js is gone.  If I want to change or tweak the production SSR server where would I do that?

**Comment by hawkeye64** [comment, marked as answer]
@bmansfie You'll want to read this: https://next.quasar.dev/quasar-cli/developing-ssr/ssr-middleware#introduction

**Reply by bmansfie** [reply]
I appreciate you taking the time to respond.

It seems that quasar is more opinionated to express.  It seems that I can no longer swap out express for another framework without completely abandoning the production system.

Are there any facilities for creating static site files from within a quasar project?

**Reply by hawkeye64** [reply]
@bmansfie SSG, no, not yet. It's on the radar.
